pkg/provider/resource_cluster.go

```diff
@@ -226,16 +226,26 @@ func (c *ClusterResource) Schema() *schema.Resource {
 							Description: "IOPS for the selected volume. It can be set to different values depending on your volume type and properties.",
 							Type:        schema.TypeString,
 							Optional:    true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								// iops is an optional field.
+								// If there is already a value set (old != "")
+								// and there is no new value (new == ""),
+								// we can suppress this Diff
+								if new == "" && old != "" {
+									return true
+								}
+								return false
```

Comment:
this could be simplified with 
```suggestion
								return new == "" && old != ""
```